frontend/app.vue

```diff
@@ -11,4 +11,8 @@
 html, body {
   @apply bg-neutral-50;
 }
+
+.Body {
+    @apply h-[calc(100vh-88px)];
+}
```

Comment:
This is a terrible hack (to hardcode height of the header in css). I suggest to remove it (here and in the other place) and instead use combination of `flex-grow` and `h-full`. Overall, you can maybe wait for the #38 merged cause it also fixes some of the related issues (eg fixed header)